js/map_utils.js

```diff
@@ -226,12 +245,23 @@ function showToolTip(d, category) {
             "Population: " + d.properties.countypop + "<br/>" +
             categoryToName(category) + ": " + 
               d.properties[[category]] + " " + "MGD");
+  if(toolTipTimer){
+    clearTimeout(toolTipTimer);
+  }
+  toolTipTimer = setTimeout(function(){
+     gtag('event', 'hover', {
+  'event_category': 'figure',
+  'event_label': d.properties.COUNTY + ", " + d.properties.STATE_ABBV + '; category=' + category + '; view=' + activeView});
+  }, toolTipDelay);
 }
 
 function hideToolTip() {
   d3.select(".tooltip")
     .classed("shown", false)
     .classed("hidden", true);
+  if (toolTipTimer){
+      clearTimeout(toolTipTimer); // stop ga for edge states
+    }
```

Comment:
not sure what you mean by "stop ga for edge states". Wouldn't it just get cleared the next time `showToolTip` is called?

Reply:
ahhhh OK